includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -153,10 +174,11 @@ public function javascript_params() {
 		$stripe_params['isCheckout']               = ( is_checkout() && empty( $_GET['pay_for_order'] ) ) ? 'yes' : 'no'; // wpcs: csrf ok.
 		$stripe_params['isOrderPay']               = is_wc_endpoint_url( 'order-pay' ) ? 'yes' : 'no';
 		$stripe_params['return_url']               = $this->get_stripe_return_url();
-		$stripe_params['ajaxurl']                  = WC_AJAX::get_endpoint( '%%endpoint%%' );
+		$stripe_params['ajaxUrl']                  = WC_AJAX::get_endpoint( '%%endpoint%%' );
 		$stripe_params['createPaymentIntentNonce'] = wp_create_nonce( '_wc_stripe_nonce' );
 		$stripe_params['upeAppeareance']           = get_transient( self::UPE_APPEARANCE_TRANSIENT );
 		$stripe_params['paymentMethodsConfig']     = $this->get_enabled_payment_method_config();
+		$stripe_params['accountDescriptor']        = 'accountDescriptor'; // TODO: this should be added to the Stripe settings page or remove it from here.
```

Comment:
Is this equivalent to "Statement Descriptor" we have on Stripe?

<img width="677" alt="Screen Shot 2021-08-02 at 7 33 47 PM" src="https://user-images.githubusercontent.com/1025173/127939820-62e1f86c-e293-47fa-be21-efdde2c85355.png">

Reply:
It's a TODO because in this PR the gateway is an independent one (called Stripe UPE), but with the changes in the admin settings, after merging the PRs it will be just an attribute of the parent class.